python_basics_loops_conditionals/python_basics_loops_conditionals.md

```diff
@@ -0,0 +1,397 @@
+
+@sage
+<div class="python">
+<lia-keep>
+<script type="text/x-sage">
+
+print(list(range(1, 20))[7])
+
+</script>
+</lia-keep>
+</div>
+
+
+### Quiz: Loops
+
+
+@sage
+<div class="python">
+<lia-keep>
+<script type="text/x-sage">
+
+for i in range(0, 6):
+  j = i*i
+</script>
+</lia-keep>
+</div>
+
+The loop in the code cell above is missing a `print()` statement to show us the output. How would you include a `print()` statement such that all of the squares of `i` in `range(0,6)` are displayed? Feel free to edit the code cell above and experiment to find the answer.
+
+[( )] `print(i)`, indented so that it **inside** the loop
+[(X)] `print(j)`, indented so that it **inside** the loop
+[( )] `print(i)` with no indentation, so that it is **outside** of the loop
+[( )] `print(j)` with no indentation, so that it is **outside** of the loop
+***
+<div class = "answer">
+
+According to our code, we know that `j` is the square of `i`, so that's the number we need to print. If we put the `print()` statement **outside** of the loop, we only see the last number, 25, because that is the value that `j` had once the loop was completed. If we place the `print()` statement **inside** the loop, each value of `j` is printed before the next iteration of the loop. Therefore, the second option is correct.
+
+</div>
+***
+
+## Conditional statements: If-else
+
+Sometimes when you're working with Python, you might want your code to do different things in different circumstances. You handle this with an `if-else`, or **conditional**, statement.
+
+Conditional statements often make use of **comparison operators**. Comparison operators compare values and return a `True` or `False` (also known as a [**boolean value**](https://www.geeksforgeeks.org/boolean-data-type-in-python/)), depending on the outcome of the comparison. Some important comparison operators include:
+
+* `==`: In Python, this is used to test **equality** (be sure not to use  the single equals sign `=`, which is used for assigning values to variables). So `9 == 18` asks the questions "is 9 equal to 18?", which would evaluate to `False`.
+
+* `!=`: not equal to.
+
+* `<`: less than.
+
+* `>`: greater than.
+
+* `<=`: less than or equal to.
+
+* `>=`: greater than or equal to.
+
+Let's look at a simple example of some code that utilizes conditionals.
+
+@sage
+<div class="python">
+<lia-keep>
+<script type="text/x-sage">
+
+name = "Pythonista"
+if type(name) == str: # We're testing here to see if the value of name is a string.
+                                    # This will either evaluate to True or False.
+    print("Welcome, "+ name+"!")
+else:
+    print("Please enter a name.")
+
+</script>
+</lia-keep>
+</div>
+
+**Your Turn**: Try changing the value assigned to `name` from "Pythonista" to a number, and see what happens! What happens if you remove the quotation marks?
+
+Multiple conditions
+======
```

Comment:
```suggestion
### Multiple conditions

```

This is a totally stylistic suggestion to make this into its own subpage so that "multiple conditions" shows up in the table of contents. Feel free to reject it :)